Formula/c/cdxgen.rb

```diff
@@ -14,6 +14,7 @@ class Cdxgen < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "307568fcf2426432b90623712c58a7f3cb5281b997722cc6aad828cb09927428"
   end
 
+  depends_on "ruby" => :test
```

Comment:
either `depends_on "ruby"` or do (as libruby is runtime dep for linux build)
```suggestion
  uses_from_macos "ruby"
```